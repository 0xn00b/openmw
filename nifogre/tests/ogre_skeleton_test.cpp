#include "ogre_mesh_common.cpp"

void C::doTest()
{
  MeshPtr msh = makeMesh("mesh1");

  // Display the mesh
  {
    SceneNode *node = mgr->getRootSceneNode()->createChildSceneNode("node");
    Entity *ent = mgr->createEntity("Mesh1", "mesh1");
    node->attachObject(ent);
    node->setPosition(0,0,4);
  }
}
