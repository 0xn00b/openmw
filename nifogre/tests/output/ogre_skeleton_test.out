Manually loading mesh mesh1
